app/common/state/ledgerState.js

```diff
@@ -344,6 +344,15 @@ const ledgerState = {
     return state.setIn(['about', 'preferences', 'updatedStamp'], date)
   },
 
+  getBackupNotifyCount: (state) => {
+    state = validateState(state)
+    return state.getIn(['about', 'preferences', 'backupNotifyCount'], 0)
```

Comment:
Please add getter to the `aboutPreferencesState.js` and we can use it directly (so that we will not have getter for every prop)